tests/lib/prepare-project.sh

```diff
@@ -68,17 +68,18 @@ fi
 
 # declare the "quiet" wrapper
 . "$TESTSLIB/quiet.sh"
+. "$TESTSLIB/dirs.sh"
```

Comment:
Agreed on the possible spread improvement but consider cluttering *every* `task.yaml` vs exactly one `spread.yaml`